Gems/Terrain/Code/Source/Components/TerrainWorldRendererComponent.cpp

```diff
@@ -85,15 +99,65 @@ namespace Terrain
                         ->Attribute(AZ::Edit::Attributes::ReadOnly, &MeshConfiguration::IsClodDisabled)
                     ;
 
+                editContext->Class<ClipmapConfiguration>("Clipmap", "Settings related to clipmap rendering")
+                    ->DataElement(AZ::Edit::UIHandlers::ComboBox, &ClipmapConfiguration::m_clipmapSize, "Clipmap image size", "The size of the clipmap image in each layer.")
+                        ->EnumAttribute(ClipmapConfiguration::ClipmapSize1024, "1024")
+                        ->EnumAttribute(ClipmapConfiguration::ClipmapSize512, "512")
```

Comment:
Very typical for games to do 2k here. Some games might even want 4k. Does this need to be fixed powers of 2? I'd think any multiple of 4 should be fine.

Reply:
It will hit the memory issue when it is increased to 2048. I'll add an 2048 option but set it later.
Technically it doesn't have to be power of 2. But it may gain some benefits when updating and dispatching compute threads.